modules/status-roles/events/presenceUpdate.js

```diff
@@ -0,0 +1,31 @@
+const {localize} = require('../../../src/functions/localize');
+
+module.exports.run = async function (client, oldPresence, newPresence) {
+
+    if (!client.botReadyAt) return;
+    if (newPresence.member.guild.id !== client.guildID) return;
+    const moduleConfig = client.configurations['status-roles']['config'];
+    const roles = moduleConfig.roles;
+    const status = moduleConfig.words;
+    const member = newPresence.member;
+
+    if (newPresence.activities.length > 0) {
+        if (newPresence.activities[0].state) {
+            if (status.some(word => newPresence.activities[0].state.toLowerCase().includes(word.toLowerCase()))) {
+                return member.roles.add(roles, localize('status-role', 'fulfilled'));
+            } else {
+                for (const key in roles) {
+                    if (member.roles.cache.has(roles[key])) {
+                        member.roles.remove(roles[key], localize('status-role', 'not-fulfilled'));
+                    }
+                }
```

Comment:
Duplicated code (24-26). Try to build a helper-function to increase maintainability.